fs/log/log.go

```diff
@@ -90,10 +90,10 @@ func Stack(o interface{}, info string) {
 func InitLogging() {
 	flagsStr := "," + Opt.Format + ","
 	var flags int
-	if strings.Contains(flagsStr, ",date,") {
+	if strings.Contains(flagsStr, ",date,") && !fs.Config.LogSystemdSupport {
```

Comment:
You've only address the default `--log-format` here.

I presume we don't want any time ever if `fs.Config.LogSystemdSupport` is set do we? In which case maybe `if` out all the time setting?

Reply:
yes no need to test it multiples times, especially after a strings contains as it is a bool (should be first test). I will fix that.